### PR TITLE
Add TG Llama70b test for APC

### PR DESCRIPTION
**Pull request by djordje-tt**
### Problem description
With lots of failures, hangs and perf regressions we spend a lot of time reverting/fixing and investigating stuff. APC test will prevent all these scenarios and give us our time back to work on other important tasks. 

### What's changed
APC test will run 80L model for prefill+decode with testing pcc of prefilled tensor logits, swapping predicted with reference token and testing pcc on 1st decode iteration. Also, testing perf and asserting if perf is lower than expected.

TODO: In this PR I will add experimentally averaged throughput yet. 

### Checklist
- [ ] [Test executed 20 times for stats retrieval](https://github.com/tenstorrent/tt-metal/actions/runs/16320357003)
- [x] [TG Demo](https://github.com/tenstorrent/tt-metal/actions/runs/16321998231)

### CODE REVIEW

**Review by sraizada-tt — APPROVED** [review]
lgtm if demo pipeline passes!
